backend/routes/auth/index.js

```diff
@@ -6,4 +6,7 @@ router.use('/linkedin', require('./linkedin'));
 /* facebook authorization */
 router.use('/facebook', require('./facebook'));
 
+/* instagram authorization */
+router.use('/instagram', require('./instagram'));
+
```

Comment:
So there will be 2 parts to Influencer auth?

Reply:
yes. first one is for facebook (need to be logged in to facebook) and the other one is for sharing IG information (no need to log in again). The second will open IG authorization agreement window as I understand. 
https://developers.facebook.com/docs/instagram-basic-display-api/getting-started